darts/models/kalman.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+import pandas as pd
+
+from abc import ABC
+
+from typing import Optional
+from filterpy.kalman import KalmanFilter
+
+from .smoothing_model import SmoothingModel
+from ..timeseries import TimeSeries
+
+
+class Kalman(SmoothingModel, ABC):
+
+    def __init__(
+            self, 
+            dim_x: int = 1, 
+            dim_z: int = 1, 
```

Comment:
Should this be called `dim_y` (following the docstring)?
Also if I understand well it should match the dimensionality of the TimeSeries it's used for, so I think it would be nice to lazily set this variable upon calling `filter(ts)` to the value of `ts.width`.

Reply:
corrected.